CONTRIBUTING.md

```diff
@@ -8,98 +8,96 @@ are important to the project's success.
 
 ## The contribution process at a glance
 
-1. Read the [README.md file](README.md).
-2. If you want, set up your environment to be able to [run tests](README.md#running-the-tests).
+1. Find out [where to make your changes](#where-to-make-changes).
+2. [Prepare your changes](#preparing-changes):
+    * Small fixes and additions can be submitted directly as pull requests,
+      but [contact us](README.md#discussion) before starting significant work.
+    * Create your stubs, considering [what to include](#what-to-include) and
+      conforming to the [coding style](#stub-file-coding-style).
+    * Reformat your stubs with `black` and `isort`.
+3. If you want, set up your environment to be able to [run tests](#running-the-tests).
     This can be useful for big pull requests or fixing specific errors, but
     usually is not needed, because the tests run automatically on GitHub Actions
     for all pull requests.
-3. [Prepare your changes](#preparing-changes):
-    * Small fixes and additions can be submitted directly as pull requests,
-      but [contact us](#discussion) before starting significant work.
-    * Create your stubs [conforming to the coding style](#stub-file-coding-style).
-    * Make sure your tests pass cleanly on `mypy`, `pytype`, and `flake8`.
-    * Reformat your stubs with `black` and `isort`.
 4. [Submit your changes](#submitting-changes) by opening a pull request.
 5. You can expect a reply within a few days:
-    * Diffs are merged when considered ready by the core team.
-    * Feel free to ping the core team if your pull request goes without
-      a reply for more than a few days.
+    * Diffs are merged when considered ready by a maintainer.
+    * Feel free to ping the [maintainers](MAINTAINERS.md) if your pull request
```

Comment:
Re: the discussion about maintainers. Most people listed here aren't particularly active on typeshed, so not sure just picking one from the list would be an effective strategy. Also "Diffs are merged when considered ready" isn't saying much and can be edited out.

Reply:
I'll reformulate and shorten it.